keyboards/sck/m0116b/keymaps/m0118/keymap.c

```diff
@@ -0,0 +1,118 @@
+/* Copyright 2018 jrfhoutx
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+ /* Keymap BASE: (Base Layer) Default Layer
+   * ,---------------------------------------------------------------------------------.
+   * |                     |  SAFE_RANGE  |                                            |
+   * `---------------------------------------------------------------------------------'
+   * ,-----------------------------------------------------------. .-------------------.
+   * |Esc| 1 |  2|  3|  4|  5|  6|  7|  8|  9|  0|  -|  =|Backsp | |MO(1)| = | /  |  * |
+   * |-----------------------------------------------------------| |-------------------|
+   * |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|Enter| | 7  | 8  | 9  | +  |
+   * |--------------------------------------------------------|  | |-------------------|
+   * |Caps   |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '| #  |  | | 4  | 5  | 6  | -  |
+   * |-----------------------------------------------------------' |-------------------|
+   * |Shift|<>|  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift| Up | | 1  | 2  | 3  | Ent|
+   * |-----------------------------------------------------------' `--------------|    |
+   * |Ctrl | Opt | Cmd |       Space        |Cmd |Left|Right|Down| | 0       | .  |    |
+   * `-----------------------------------------------------------' `-------------------'
+   */
+[0] = LAYOUT_m0118_iso( /* Base */
+															                           SAFE_RANGE, 
```

Comment:
```suggestion
															                           M0116B, 
```

Reply:
None of the keymaps  compile when using `M0116B` instead of `SAFE_RANGE`. they all compile perfectly fine, and the sendstring works correctly, when using `SAFE_RANGE`, which is why that is what is in in those spaces

Reply:
corrected